### ISSUE TITLE
Not working with Chrome 88

### ISSUE THREAD

**Issue by kunte0** [tomer8007/chromium-ipc-sniffer#1]
.\chromeipc.exe --update-interfaces-info
```
[+] 1002.Processing file ui/ozone/public/mojom/scenic_gpu_host.mojom
[+] 1003.Processing file ui/ozone/public/mojom/scenic_gpu_service.mojom
[+] 1004.Processing file ui/ozone/public/mojom/wayland/wayland_buffer_manager.mojom
[+] 1005.Processing file ui/ozone/public/mojom/wayland/wayland_overlay_config.mojom
[+] 1006.Processing file ui/webui/resources/cr_components/customize_themes/customize_themes.mojom
[+] 1007.Processing file url/mojom/origin.mojom
[+] 1008.Processing file url/mojom/url.mojom
[+] 1009.Processing file url/mojom/url_test.mojom
[+] Matching commit is 2489b0741310c3a50c990c1d6acb9f113bd879a8
[+] Going to download 30 legacy IPC header files.
[+] Processing file content/common/frame_messages.h
[+] Processing file content/common/page_messages.h

Unhandled Exception: System.Net.WebException: The remote server returned an error: (404) Not Found.
   at ChromeIPCSniffer.LegacyIpcInterfacesFetcher.DownloadAndAnalyzeLegacyIpcFiles(Dictionary`2 legacyIpcFiles, String commit, String chromeVersion)
   at ChromeIPCSniffer.LegacyIpcInterfacesFetcher.UpdateInterfacesInfoIfNeeded(String chromeVersion, Boolean force)
   at ChromeIPCSniffer.Program.Main(String[] args)
```
Running .\chromeipc.exe alone now chashes:
```
Chrome IPC Sniffer v0.5.0.0

Type -h to get usage help and extended options

[+] Starting up
[+] Determining your chromium version
[+] You are using chromium 88.0.4324.150
[+] Checking mojom interfaces information
[+] Checking legacy IPC interfaces information

Unhandled Exception: Newtonsoft.Json.JsonReaderException: Unterminated string. Expected delimiter: ". Path '101DB.link', line 59, position 98.
   at Newtonsoft.Json.JsonTextReader.ReadStringIntoBuffer(Char quote)
   at Newtonsoft.Json.JsonTextReader.ParseValue()
   at Newtonsoft.Json.JsonWriter.WriteToken(JsonReader reader, Boolean writeChildren, Boolean writeDateConstructorAsDate, Boolean writeComments)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateJObject(JsonReader reader)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonConvert.DeserializeObject(String value, Type type, JsonSerializerSettings settings)
   at ChromeIPCSniffer.LegacyIpcInterfacesFetcher.UpdateInterfacesInfoIfNeeded(String chromeVersion, Boolean force)
   at ChromeIPCSniffer.Program.Main(String[] args)
```

**Comment by tomer8007** [comment]
Thanks, this bug happened because of the ongoing process in chromium to remove legacy IPC interfaces, in this case due to [this commit](https://github.com/chromium/chromium/commit/c088829b7e4576fe9d8aae19782d5abe86a810c3) (removal of `page_messages.h`).

I fixed it in 733e45472cf75ff8b914206942a54b48d8066a17.

**Comment by tomer8007** [comment]
Also this shows a more general problem, we'll need to make the enum update dynamically based on the chromium version, to avoid changes such as c7da9782f8593a28dbe30d4c1170be4796086025.
